infrastructure/lib/stacks/monitoringDashboard.ts

```diff
@@ -69,6 +69,7 @@ export class OpenSearchMetricsMonitoringStack extends Stack {
     private snsMonitorStepFunctionExecutionsFailed(): void {
         const stepFunctionSnsAlarms = [
             { alertName: 'StepFunction_execution_errors_MetricsWorkflow', stateMachineName: this.props.workflowComponent.opensearchMetricsWorkflowStateMachineName },
+            { alertName: 'StepFunction_execution_errors_MaintainerInactivityWorkflow', stateMachineName: this.props.workflowComponent.opensearchMaintainerInactivityWorkflowStateMachineName },
```

Comment:
NIT: moving forward should this `infrastructure/lib/stacks/monitoringDashboard.ts` have inputs on the workflows to monitor?